tests/app/item/CDTest.java

```diff
@@ -0,0 +1,42 @@
+package app.item;
+
+import app.catalogue.Catalogue;
+import app.catalogue.CataloguePopulator;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * This class will test the functionality of the toString method for the CD object.
+ *
+ * @author chad_olsen
+ * @since 2017/06/28.
+ */
+public class CDTest {
+    @Test
+    public void toStringTest() throws Exception {
+        CD cd = new CD("Now 45", "Pop", "1hr30mins", 12);
+        cd.addArtist("Bob");
+        cd.addArtist("Sam");
+
+        Catalogue catalogue = CataloguePopulator.populateCatalogue();
+        String catalogueString = catalogue.getCatalogueMap().get(Type.CD).get(0).toString();
+        assertEquals(catalogueString, cd.toString());
+
+        CD newCD = new CD("Rock Hits 45", "Rock", "1hr30mins", 0);
+        catalogue.addItemToList(newCD);
+        catalogueString = catalogue.getCatalogueMap().get(Type.CD).get(2).toString();
+        assertEquals(catalogueString, newCD.toString());
+    }
+
+    @Test
+    public void toString_nullTest() {
+        CD cd = new CD();
+
+        Catalogue catalogue = new Catalogue();
+        catalogue.addItemToList(cd);
+        String catalogueString = catalogue.getCatalogueMap().get(Type.CD).get(0).toString();
+        assertEquals(catalogueString, cd.toString());
```

Comment:
You are comparing the toString() values from the exact same object, they were just retrieved differently. Rather build up the string manually and then compare it to the value returned on the CD object so that we can ensure the value hasn't changed or this test would never ever fail.